src/pg_to_evalscript/evalscript.py

```diff
@@ -131,13 +151,21 @@ def write_update_output(self):
 
     def write_output_variable(self):
         if len(self.nodes) == 0:
-            return self.initial_data_name
+            return "_".join(self.initial_data_names.values())
         return self.nodes[-1].node_varname_prefix + self.nodes[-1].node_id
 
     def determine_output_dimensions(self):
         dimensions_of_inputs_per_node = defaultdict(list)
+        all_bands = []
+        for datasource_with_bands in self.input_bands:
```

Comment:
Can we have some comment somewhere how one element (`datasource_with_bands`) of `self.input_bands` looks like because even now, I'm constantly not sure.

Reply:
You can ignore the above comment, I got confused because of the way the `all_bands.extend()` is written.

I think it would be clearer if it would be like:

```python
        for datasource_with_bands in self.input_bands:
            if datasource_with_bands is not None and datasource_with_bands["bands"] is not None:
                all_bands.extend(datasource_with_bands["bands"])

```

Reply:
Changed

Reply:
Thanks